vendor/vendor.json

```diff
@@ -964,10 +964,10 @@
 			"revisionTime": "2016-12-14T09:25:55Z"
 		},
 		{
-			"checksumSHA1": "FEzQdhqmb6aqGL1lKnjOcUHIGSY=",
+			"checksumSHA1": "jhyv7qysp0NZbWzAaUCsKATDGuk=",
 			"path": "google.golang.org/api/bigquery/v2",
```

Comment:
How is updating bigquery client to the latest version related to this change?

Reply:
When I updated storage, it didn't build because it relied on a version of gensupport later than what we currently had vendored. When I updated gensupport, it still didn't build because bigquery was relying on an older version of gensupport.